### PR TITLE
Removed all //ok comments

### PR DESCRIPTION
**Pull request by Shobhit3205**
# PLEASE READ before removing

Rules:
1) The issue you are trying to fix/resolve has to have the "approved" label.
2) Put in the description of Pull Request the reference to an issue if it exists.
   Example: "Issue: #XXXXXX"
3) Commit message should adhere to the following rules:
   a) MUST match one of the following patterns:
      ^Issue #\\d+: .*$
      ^Pull #\\d+: .*$
      ^(minor|config|infra|doc|spelling|dependency): .*$
   b) MUST contain only one line of text
   c) MUST NOT end with a period, space, or tab
   d) MUST be less than or equal to 200 characters

To avoid multiple iterations of fixes and CIs failures, please read
https://checkstyle.org/contributing.html

ATTENTION: We are not merging Pull Requests that are not passing our CIs,
but we will help to resolve issues.

Thanks for reading, remove whole this message and type what you need.

### CODE REVIEW

**Comment by Shobhit3205** [comment]
removed //ok from the code

File: .ci/diff-report.sh [hunk 1]
```diff
@@ -95,6 +95,11 @@ parse-pr-description-text)
   NEW_MODULE_CONFIG_LINK=$(echo "$NEW_MODULE_CONFIG_PARAMETER" | sed -E 's/New module config: //')
   PATCH_CONFIG_LINK=$(echo "$PATCH_CONFIG_PARAMETER" | sed -E 's/Diff Regression patch config: //')
   REPORT_LABEL=$(echo "$REPORT_LABEL_PARAMETER" | sed -E 's/Report label: //')
+  # trim
```
Comment (MANISH-K-07) [inline .ci/diff-report.sh:98]: The diff in your PR is of 925 commits and 3671 changed files. The branch head amust have got updated with merge commits. Please reset your he

**Review by MANISH-K-07 — COMMENTED** [review]
- Read the PR template, make sure you understand the template, edit the PR description -> remove the template -> update the description as specified in the template.
- Please remove comments for all the files concerning that module in this PR and update the commit message, PR title accordingly.
- Suppression for the corresponding comments should be also removed from the [file](https://github.com/checkstyle/checkstyle/blob/b6301c189ef5d2d731a755fa9a2e84aa0677494a/config/checkstyle-input-suppressions.xml#L68-L70), after the removal of comments.
- Please go through the issue description and the merged PRs for a better understanding regarding the changes.
- Make sure mvn clean verify passes locally before pushing the changes.
- Keep this PR to one commit only, squash any additional commits. ([Reference](https://checkstyle.org/beginning_development.html#Starting_Development))

Go through the [documentation](https://checkstyle.org/beginning_development.html) before you get started to get familiarized with the project.

**Comment by nrmancuso** [comment]
@Shobhit3205 please carefully read through https://checkstyle.org/beginning_development.html.

It is advisable to never open a PR from your fork's master branch. Please create a new PR, following the documentation linked above, that contains only changes that relate to the linked issue.

**Pull request closed without merging**